### PR TITLE
Add support for route flow counter

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
What I did?

Add new table names for route flow counter feature.

HLD:
https://github.com/Azure/SONiC/pull/908

### CODE REVIEW

**Review by keboliu — APPROVED** [review]

**Comment by Junchao-Mellanox** [comment]
Hi @prsunny , could you please review this? It is a small change which only contains new schema definitions.

File: common/schema.h [hunk 1]
```diff
@@ -370,6 +372,8 @@ namespace swss {
 
 #define CFG_DHCP_TABLE                              "DHCP_RELAY"
 
+#define CFG_FLOW_COUNTER_ROUTE_PATTERN_TABLE        "FLOW_COUNTER_ROUTE_PATTERN_TABLE"
```
Comment (prsunny) [inline common/schema.h:375]:
```suggestion
#define CFG_FLOW_COUNTER_ROUTE_PATTERN_TABLE        "FLOW_COUNTER_ROUTE_PATTERN"
```

**Review by prsunny — APPROVED** [review]

**Comment by Junchao-Mellanox** [comment]
Hi @prsunny , could you please help re-trigger the test? Thanks.

**Comment by Junchao-Mellanox** [comment]
Did a rebase, hope it could fix the vstest issue.

**Review by prsunny — APPROVED** [review]

**Pull request merged**